src/components/VillagerTypesList/VillagerTypesList.tsx

```diff
@@ -5,7 +5,7 @@ import Villager from '../../types/Villager';
 import './VillagerTypesList.css'
 
 export interface Props {
-  allVillagers: Villager[];
+  allVillagers: {[species: string]: Villager[]};
```

Comment:
I'm not super familiar with typescript but I love to see that you are using a typelist file! It is great organization!